qek/data/extractors.py

```diff
@@ -947,17 +947,16 @@ def __init__(
             username=username,
             device_name=device_name,
             password=password,
-            batch_ids=batch_ids,
+            job_ids=job_ids,
             path=path,
         )
 
     def run(self, dt: int = 10) -> PasqalCloudExtracted:
         def extractor(job: Job, sequence: pl.Sequence) -> dict[str, int] | None:
-            cutoff_duration = int(ceil(sequence.get_duration() / dt) * dt)
             full_result = job.full_result
             if full_result is None:
                 return None
-            result = full_result["bitstring"][cutoff_duration]
+            result = full_result["counter"]
```

Comment:
The cloud MPS integration has been updated recently to be consistent with other backends; results are now properly stored in the "counter" attribute. Also for now, only the bitstring at the last time step are available and the counter is stored directly in that dictionary value

Reply:
For our usecase, that's great!

Do you have plans to upgrade this (for users who might need more observables) without breaking compatibility?

Reply:
We are working on letting user configure the backend results/observables for remote backends (joint work with the pulser/emulator team), it should available in the coming weeks or months. We have not fleshed out the details of the interface but we'll try to enforce backwards compatibility as much as possible